.gitignore

```diff
@@ -4,4 +4,6 @@ bin/
 *.o
 .vscode/
 *.dll
-*.exe
\ No newline at end of file
+*.exe
+/cmake-build-release/
```

Comment:
using CLion? What do you think about placing such folders inside a `build` directory at project root? This way we don't need to create an entry here to every cmake profile

Reply:
Ah, yes. True. Will try to do it